src/prisma/schema.prisma

```diff
@@ -38,3 +40,16 @@ model Profile {
   website                  Json?
   tdaGradYearBannerVisible Json
 }
+
+model Post {
+  id                 Int   @id @default(autoincrement())
+  userId             Int?   @unique
+  user               User?  @relation("UserToPost",fields: [userId], references: [id])
+  content            String
+  image              String?
+  likes              Int?      @default(0)
+  dislikes           Int?      @default(0)
+  isDraft            Boolean?
+  hasSensitiveTopic  Boolean?
+  // privacyOption      PrivacyOption?
```

Comment:
I think we can name this field `visibleTo` so that we stay in line with what we are calling it on profile. And I would keep it as a required field with the default setting as community.